nx.json

```diff
@@ -101,6 +101,22 @@
     },
     "remix-ui-checkbox": {
       "tags": []
+    },
+    "compiler-artefacts": {
```

Comment:
Add library names to lint command in package.json https://github.com/ethereum/remix-project/blob/master/package.json#L44

Reply:
ok